tests/backup/cluster_controller_test.go

```diff
@@ -0,0 +1,23 @@
+package tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/portworx/torpedo/drivers/backup/cluster_controller"
+	"github.com/portworx/torpedo/drivers/backup/cluster_controller/cluster_utils"
+	"github.com/portworx/torpedo/pkg/log"
+	"time"
+)
+
+var _ = Describe("{ClusterControllerTest}", func() {
```

Comment:
This test case is not meant to be merged but to serve as a demo, hopefully making it convenient for reviewers to understand the functionalities of the ClusterController.